### PR TITLE
fix bug caused by without close stmt

### PR DESCRIPTION
**Pull request by GeorgeXc**
This commit add a function to fix the bug reported in the pull request https://github.com/astaxie/beego/pull/3825/commits/330e4ef970c2bae454bdb17823c5d7485a461e17.

To make this fix working .
we need add a line before our program exit
`orm.DeregisterAllDatabases()`

### CODE REVIEW

**Comment by GeorgeXc** [comment]
@astaxie 
This is an emergency bug fix. And we also need change file https://github.com/beego/beedoc/blob/master/zh-CN/mvc/model/orm.md. 
And I have create an pull request for this https://github.com/GeorgeXc/beedoc/pull/1

**Comment by GeorgeXc** [comment]
@astaxie

**Comment by GeorgeXc** [comment]
@astaxie

**Comment by GeorgeXc** [comment]
@astaxie

File: orm/README.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ func init() {
 }
 
 func main() {
-	o := orm.NewOrm()
+    o := orm.NewOrm()
+    defer orm.DeregisterAllDatabases()
```
Comment (astaxie) [inline orm/README.md]: This is a break API

File: orm/db_alias.go [hunk 2]
```diff
@@ -98,6 +98,20 @@ func (ac *_dbCache) get(name string) (al *alias, ok bool) {
 	return
 }
 
+func (ac *_dbCache) removeAllAlias() {
+	ac.mux.Lock()
+	defer ac.mux.Unlock()
+	for _, al := range ac.cache {
+		al.DB.Lock()
+		for _, stmt := range al.DB.stmts {
+			stmt.Close()
```
Comment (astaxie) [inline orm/db_alias.go]: Why not call `stmt.Close` when use it?

Comment (GeorgeXc) [reply orm/db_alias.go]: stmt实际上是在数据库服务器端对sql语句的预编译，下次执行再次使用这个stmt的时候不需要把sql语句传给服务端，只需要传sql语句中的参数即可。如果每次都传整个sql语句给数据库服务器，将会加大数据库服务器的压力。

File: orm/README.md (hunk 1 above)
Comment (GeorgeXc) [reply orm/README.md]: Yes , It is.

**Pull request closed without merging**